classes_part1.py

```diff
@@ -0,0 +1,61 @@
+import datetime
+import time
+import pet
+
+# Create empty array to store instances of pets in
+pet_list = []
+
+# Get info about Pet 1 and store it
+print("Let's start by gathering basic data about your first pet.\n")
+pet_1 = pet.pet('','','')
```

Comment:
I see that you put your logic to get user input into the methods "getName", "getCategory", "getSubtype".  I would recommend not doing this.  Additionally, the general naming convention for methods in a class is that the "get-----" would be so you can access the data.  You would not be "getting" user input with a "get" method name.

You are creating the class with nothing, and this defeats the purpose of having requires parameters in the class constructor.  You've found a way around the functionality :)

1. Move the "blah = input" out of the class methods and put it in your classes_part1.py
2. Rename your methods that store the values to "setName", "setCategory", etc.
3. Do not call the constructor and pass in temporary null strings.  You are not even using the constructor for it's purpose at this point

**instead do it like this:**

```python

# Gather the name, category, and subcategory somehow
pet_1 = pet.pet(temp_name, temp_category, temp_subcategory)
pet_list.append(pet_1)

# Gather all data in temporary variables (I've just prefixed them with t_ to demonstrate, name them however
pet_2 = pet.pet(t_name, t_category, t_subcategory, t_birthdate, t_colors, t_has_tail, t_has_wings, t_num_of_legs, t_last_fed, t_deathdate)
pet_list.append(pet_2)
```